configure.ac

```diff
@@ -994,6 +996,12 @@ AM_COND_IF([ENABLE_INIT], [build_init="${init_dir}"], [build_init=no])
 
 AM_COND_IF([ENABLE_SYSTEMD], [build_systemd="${systemd_dir}"], [build_systemd=no])
 
+#
+# FreeRDP plugins
+#
+
+AM_COND_IF([ENABLE_RDP], [build_rdp_plugins="${FREERDP2_PLUGIN_DIR}"], [build_rdp_plugins=no])
```

Comment:
What happens if this ends up no or undefined - because FreeRDP2, for whatever reason, lacks the plugins directory?

Reply:
It depends on which variable you're referring to:

The `build_rdp_plugins` variable is only used to represent the status of the build visually within the summary. If it ends up being "no" (due to FreeRDP being absent or due to RDP being explicitly disabled via command line options passed to `configure`), then the summary ends up looking like:

```
...
   Services / tools:

      guacd ...... yes
      guacenc .... yes
      guaclog .... yes

   FreeRDP plugins: no
   Init scripts: no
   Systemd units: no
...
```

`FREERDP2_PLUGIN_DIR` cannot be "no". It could potentially be a strange value if the pkg-config file installed by FreeRDP is broken, in which case the RDP part of the build would be broken.

I'll add a `--with-freerdp-plugin-dir` option to override the auto-detected value, in case the detected value ends up being incorrect.

Reply:
OK, I've added a `--with-freerdp-plugin-dir` option. From `./configure --help`:

```
...
  --with-terminal         support text-based protocols [default=check]
  --with-vnc              support VNC [default=check]
  --with-rdp              support RDP [default=check]
  --with-freerdp-plugin-dir=<path>
                          install FreeRDP plugins to the given directory
                          [default=check]
...
```

If given, the value will be used instead of attempting to derive the value from the package lib directory exposed by pkg-config.